orca-front50/orca-front50.gradle

```diff
@@ -19,6 +19,9 @@ apply from: "$rootDir/gradle/groovy.gradle"
 dependencies {
   implementation(project(":orca-core"))
   implementation(project(":orca-retrofit"))
+
+  api("org.codehaus.groovy:groovy")
```

Comment:
Does this really need to be API? I don't really see anything that needs to be exported, but maybe there are things in the bytecode?

Reply:
Yeah, I added a comment above in the PR message...but when you try to consume the `Application` class it says that the superclass (which is a groovy object) is not on the classpath.

Reply:
> Yeah, I added a comment above in the PR message...

Whoops, so you did.